### ISSUE TITLE
Precompiled manifest not rendering referenced external script

### ISSUE THREAD

**Issue by jgeurts** [andrewdavey/cassette#195]
I have a js file that references an external script.  When I use the precompiled manifest file from Cassette.MSBuild, the referenced external script is not getting included in my rendered output.  If I remove the precompiledManifest attribute from the cassette element and set debug="false", then the external script is included.

My script file looks something like:

```
/// <reference path="~/Content/js/Shared/Site.js" />
/// <reference path="//use.typekit.com/asdf.js" />

try{Typekit.load();}catch(e){}
```

**Comment by andrewdavey** [comment]
My initial guess is that the serialized manifest is missing the external reference. Or the manifest reader is skipping references when deserializing.

**Comment by jgeurts** [comment]
I looked at the manifest and the external reference is there

**Comment by andrewdavey** [comment]
Ah yes, just checked that as well. The issue could be in https://github.com/andrewdavey/cassette/blob/master/src/Cassette/CassetteApplicationContainerFactoryBase.cs#L58 it's probably missing the step where ad-hoc external references are converted into `ExternalScriptBundle` objects.
I'll get this fixed soon.

**Comment by andrewdavey** [comment]
The above commit seems to work for me. Can you give it a try as well please? If it's all good I'll do another release.

**Comment by jgeurts** [comment]
That seems to have fixed it for me as well.  Thanks!

**Comment by andrewdavey** [comment]
Available in v1.2.0
